build.gradle

```diff
@@ -26,7 +26,7 @@ repositories {
 dependencies {
     compile gradleApi()
     compile "org.codehaus.groovy:groovy-all:2.3.10"
-    compile 'com.android.tools.build:gradle:1.2.3'
```

Comment:
Why `2.3.3`? I think this is old version. The newest is `3.1.0`. Plz check https://developer.android.com/studio/releases/gradle-plugin